bin/meta-git-remote

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const program = require('commander');
+const debug = require('debug')('meta-git-remote');
+const loop = require('loop-things');
+const path = require('path');
+const util = require('util');
+
+const usages = `\n  usages:\n
+    meta git remote
+    meta git remote -v
+    meta git remote show origin
+    meta git remote rename oldname newname
+    meta git remote rm name`;
+
+if (process.argv[2] === '--help')
+  return console.log(usages);
+
+let first = process.argv[2] === 'blank' ?
+                      process.argv[3] || '':
+                      process.argv[2] || '';
+
+const subCommandParsers = {
+  '-v': function (argv) { return '' },
```

Comment:
hey @mateodelnorte shouldn't this line be return '-v'
`git remote` and `git remote -v` do return different output.